src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -608,121 +644,16 @@ private static bool KeepAllItemsInTheList(CompletionTriggerKind initialTriggerKi
             ImmutableArray<string> recentItems,
             bool highlightMatchingPortions,
             ref int currentIndex,
-            out MatchResult matchResult)
+            out MatchResult<VSCompletionItem> matchResult)
         {
             var roslynItem = GetOrAddRoslynCompletionItem(item);
-
-            // Get the match of the given completion item for the pattern provided so far. 
```

Comment:
As above, this code was moved down a layer.  the only meaningful change was removing the call to GetHighlightedSpans when creating the MatchResult (it's done later)

Reply:
what's the benefit of doing it later?